bitcoind-rpc/src/main/scala/org/bitcoins/rpc/client/v20/BitcoindV20RpcClient.scala

```diff
@@ -62,6 +63,36 @@ class BitcoindV20RpcClient(override val instance: BitcoindInstance)(implicit
 
   override def getFilterCount: Future[Int] = getBlockCount
 
+  override def getFilterHeaderCount(): Future[Int] = getBlockCount
+
+  override def getFilter(
+      hash: DoubleSha256DigestBE): Future[Option[CompactFilterDb]] = {
+    for {
+      header <- getBlockHeader(hash)
+      filter <- getBlockFilter(hash, FilterType.Basic)
+    } yield Some(filter.filterDb(header.height))
+  }
+
+  override def getFiltersAtHeight(
+      height: Int): Future[Vector[CompactFilterDb]] = {
+    for {
+      hash <- getBlockHash(height)
+      filter <- getBlockFilter(hash, FilterType.Basic)
+    } yield Vector(filter.filterDb(height))
+  }
+
+  // Cannot get the filter headers from the RPC
+
+  override def getFilterHeadersAtHeight(
+      height: Int): Future[Vector[CompactFilterHeaderDb]] = FutureUtil.emptyVec
+
+  override def getBestFilterHeader(): Future[Option[CompactFilterHeaderDb]] =
```

Comment:
Doesn't bitcoind v20 support filters?

Reply:
While they might not have a direct rpc for query `getbestfiltereheader` we can synthetically implement this with a call to `getbestheader` can't we? and then retrieving that filter from bitcoind?

Reply:
We can't get filter headers from the rpc

Reply:
Did they remove `getblockfilter` in v20? Otherwise can't you emulate

https://github.com/bitcoin-s/bitcoin-s/blob/fa388c744847081788a98352c304a377e815904c/testkit/src/main/scala/org/bitcoins/testkit/chain/SyncUtil.scala#L38-L37

Reply:
Wow I don't see it.... why did they get rid of it? :/ 

https://developer.bitcoin.org/reference/rpc/index.html#blockchain-rpcs

Reply:
There is still `getFilter` but there is no `getFilterHeader`

Reply:
Ok I think you've convinced me it can't be done. 

So can we just error on these cases too? Silent failures lead to hours of debugging :-)